helpers/deb.sh

```diff
@@ -13,6 +13,55 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+# Declares the DEBOOTSTRAP_EXEC environment variable which then can be used to
+# execute debootstrap.
+# Globals:
+#     DEBOOTSTRAP_DIR
+#     DEBOOTSTRAP_EXEC
+#     DEBOOTSTRAP_VER
+# Arguments:
+#     None
+# Returns:
+#     None
+init_debootstrap() {
+    local path_to_debootstrap="${PIEMAN_DIR}/${TOOLSET_DIR}/debootstrap"
+    local ver=""
+
+    DEBOOTSTRAP_EXEC="env DEBOOTSTRAP_DIR=${path_to_debootstrap} ${path_to_debootstrap}/debootstrap"
+
+    info "using ${DEBOOTSTRAP_EXEC}"
+}
+
+# Checks if debootstrap is equal to or higher than the version specified in the
+# DEBOOTSTRAP_VER environment variable.
+# Globals:
+#     DEBOOTSTRAP_VER
+#     PIEMAN_DIR
+#     TOOLSET_DIR
+# Arguments:
+#     None
+# Returns:
+#     Boolean
+is_debootstrap_uptodate() {
+    local path_to_debootstrap="${PIEMAN_DIR}/${TOOLSET_DIR}/debootstrap"
```

Comment:
Do we need PIEMAN_DIR? Seems TOOLSET_DIR is enough. Otherwise need to add a note to TOOLSET_DIR that it cannot be an absolute path.